ftw/contacts/tests/test_hide_contact_images.py

```diff
@@ -0,0 +1,46 @@
+from ftw.builder import Builder
+from ftw.builder import create
+from ftw.contacts.testing import FTW_CONTACTS_FUNCTIONAL_TESTING
+from ftw.testbrowser import browsing
+from plone.app.testing import TEST_USER_ID
+from plone.app.testing import setRoles
+from unittest2 import TestCase
+
+
+class TestHideContactImages(TestCase):
+
+    layer = FTW_CONTACTS_FUNCTIONAL_TESTING
+
+    def setUp(self):
+        self.portal = self.layer['portal']
+        setRoles(self.portal, TEST_USER_ID, ['Manager'])
+
+    @browsing
+    def test_contact_images_are_hidden_when_hide_images_checkbox_is_active(self, browser):
+        contactfolder = create(
+            Builder('contact folder').titled(u'Contact folder').having(
+                hide_contacts_image=True))
+
+        contact = create(Builder('contact')
+                         .with_maximal_info(u'Chuck', u'Norris')
+                         .within(contactfolder))
+
+        browser.login().visit(contact, view='contact_summary')
+
+        self.assertEqual(1, len(browser.css('div.contactSummary')))
+        self.assertEqual(0, len(browser.css('a.contactImage')))
+
+    @browsing
+    def test_contact_images_are_shown_when_hide_images_checkbox_is_deactivated(self, browser):
+        contactfolder = create(
+            Builder('contact folder').titled(u'Contact folder').having(
+                hide_contacts_image=False))
```

Comment:
> When applying a PEP8 guideline would make the code less readable, this particular guideline should be ignored.

Source: https://www.python.org/dev/peps/pep-0008/#a-foolish-consistency-is-the-hobgoblin-of-little-minds

My question: does this line break make the more or less readable?